gmm/sm.go

```diff
@@ -126,6 +127,10 @@ func Registered(state *fsm.State, event fsm.EventType, args fsm.ArgsType) {
 		logger.GmmLog.Debugln(event)
 	case InitDeregistrationEvent:
 		logger.GmmLog.Debugln(event)
+	case NwInitiatedDeregistrationEvent:
```

Comment:
when does amf send this event? i haven't seen code to send this event in this PR.